docs/Getting-started/FAQ.md

```diff
@@ -485,7 +485,7 @@ No. Currently, Fleet is only available for self-hosting on premises or in the cl
 
 ### What MySQL versions are supported?
 
-Fleet is tested with MySQL 5.7.21 and 8.0.28. Newer versions of MySQL 5.7 and MySQL 8 typically work well. AWS Aurora requires at least version 2.10.0. Fleet has been known to run successfully on MariaDB and other MySQL flavors but is not officially supported.
+Fleet is tested with MySQL 5.7.21 and 8.0.28. Newer versions of MySQL 5.7 and MySQL 8 typically work well. AWS Aurora requires at least version 2.10.0. We recommend against using MariaDB and other MySQL variants that are not officially supported.
```

Comment:
You may want to reconsider the beginning of the second sentence. People might miss "against" in "recommend against." Here are a couple alternatives:

"We do not recommend using MariaDB..."
"Please do not use MariaDB..."